Added row, sio and vr I/O Schedulers

DwaineGarden opened the pull request:
Adding some additional I/O Schedulers.

DwaineGarden committed:
Added row, sio and vr I/O Schedulers

DwaineGarden committed:
Fix the description in the RDS Driver Makefile

DwaineGarden commented:
Fixed the description in the Makefile for the RDS driver.

DwaineGarden committed:
Add Adaptive, Hyper, Lionheart, Lulzactive, OnDemandX and SmartAss2 governors.

DwaineGarden committed:
Enable NTFS file system module with write option.

DwaineGarden committed:
Update rk_serial.c

Implement the RDS functionality

DwaineGarden committed:
Update rk_serial.c

Fixed typo in define for POINT_SYS_VOL

DwaineGarden committed:
Update Makefile

Added missing bonovo_rds so the driver will compile.

Zaphod-Beeblebrox commented:
I just realized you did a pull request against my repo rather than the Nu3001 repo.  Probably need to fork/pull from there instead.
I did try to build this and got some errors, but I don't recall what they were.

DwaineGarden committed:
Add TCP congestion control Westwood

*Add the TCP congestion control Westwood.
*Make TCP congestion control Westwood the default.

DwaineGarden commented:
Added Westwood TCP congestion control in the kernel.  Also made it the default for the radio wifi.
